Decred WalletUITests/RestoreExistingWalletUITest.swift

```diff
@@ -0,0 +1,452 @@
+//
+//  RestoreExistingWalletUITest.swift
+//  Decred WalletUITests
+//
+// Copyright (c) 2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import XCTest
+
+class RestoreExistingWalletUITest: XCTestCase {
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+
+        // In UI tests it is usually best to stop immediately when a failure occurs.
+        continueAfterFailure = false
+
+        // In UI tests it’s important to set the initial state - such as interface orientation - required for your tests before they run. The setUp method is a good place to do this.
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    func testRestoreExistingWalletWithPassword() throws {
+        
+        let app = XCUIApplication()
+        app.launchArguments.append("--UITests")
+        app.launch()
+        
+        let permission = app.alerts["“Decred Wallet Testnet” Would Like to Send You Notifications"].scrollViews.otherElements.buttons["Allow"]
+        
+        if permission.exists {
+            permission.tap()
+        }
+        
+        let restoreExistingWallet = app.buttons.element(matching: .button, identifier: "restoreExistingWallet")
+        if restoreExistingWallet.waitForExistence(timeout: 5) {
+            restoreExistingWallet.tap()
+        }
+        
+        sleep(1)
+        
+        self.seedUIInput(app: app)
```

Comment:
false when keyboard is not in English

Reply:
this is a tests the default language and hard coded strings are in english.